service/src/main/java/bio/terra/policy/service/pao/graph/Walker.java

```diff
@@ -0,0 +1,227 @@
+package bio.terra.policy.service.pao.graph;
```

Comment:
i'd much prefer we have test coverage for Walker

Reply:
Yes, I am working on that as a separate ticket. I want to get this in so Mike and I can work in parallel. As you can see, there are a lot of cases to test in the walker and it will take awhile to create and debug those.